generators/firechip/src/main/scala/TargetConfigs.scala

```diff
@@ -179,10 +179,10 @@ class SupernodeFireSimRocketConfig extends Config(
   new FireSimRocketConfig)
 
 //**********************************************************************************
-//* Ariane Configurations
+//* CVA6 Configurations
 //*********************************************************************************/
-class FireSimArianeConfig extends Config(
+class FireSimCva6Config extends Config(
```

Comment:
This requires changing the default build recipes defined in FireSim: 
https://github.com/firesim/firesim/blob/master/deploy/sample-backup-configs/sample_config_build_recipes.ini

Reply:
OK.  

I think the whole thing requires changing the wrapper for Ariane, as well, right?

Reply:
Yes, that submodule would need to be updated and bumped as well. Our standard routine is to open PRs for and submodules which need to be bumped, and link to them in this PR. Then we merge everything together once everything is approved.